Tests/MacroToolkitTests/MacroToolkitTests.swift

```diff
@@ -411,8 +422,14 @@ final class MacroToolkitTests: XCTestCase {
             /abc/
             """
 
-        // TODO: Figure out a more precise way to test regex literal parsing
-        XCTAssert((try? RegexLiteral(basicLiteral)?.regexValue()) != nil)
+        assertInlineSnapshot(of: RegexLiteral(basicLiteral), as: .description) {
```

Comment:
You need to have `.regexValue()` still, otherwise the test isn't testing anything other than the internal structure of `RegexLiteral` (the initialiser doesn't do any parsing)